wca-guidelines.md

```diff
@@ -131,6 +131,9 @@ To be more informative, each Guideline is classified using one of the following
 - 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a cutoff round), they have no result for the attempt.
 - 9p2+) [CLARIFICATION] Result refers to the single or average that determines the ranking of the corresponding round.
 - 9q+) [RECOMMENDATION] Events and rounds should have at least 2 competitors.
+- 9t+) [EXPLANATION] Because of [9t1](regulations:regulain:9t1) and [9t2](regulations:regulain:9t2), rounds may look like they don't comply with [9p1](regulations:regulation:9m1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) or [9m3](regulations:regulation:9m3). However, makings changes to comply to these rules would affect innocent competitors, which is why the results must stay the same.
```

Comment:
```suggestion
- 9t+) [EXPLANATION] Because of [9t1](regulations:regulain:9t1) and [9t2](regulations:regulain:9t2), rounds may look like they don't comply with [9p1](regulations:regulation:9m1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) or [9m3](regulations:regulation:9m3). However, making changes to comply to these rules would affect innocent competitors, which is why the results must stay the same.
```